packages/client/tests/functional/default-selection/_matrix.ts

```diff
@@ -1,26 +1,4 @@
 import { defineMatrix } from '../_utils/defineMatrix'
+import { allProviders } from '../_utils/providers'
 
-export default defineMatrix(() => [
-  [
-    {
-      provider: 'sqlite',
-    },
-    {
-      provider: 'postgresql',
-    },
-    {
-      provider: 'mysql',
-    },
-    {
-      provider: 'mongodb',
-    },
-    {
-      provider: 'cockroachdb',
-    },
-    {
-      provider: 'sqlserver',
-    },
-  ],
-
-  [{ previewFeatures: '' }, { previewFeatures: '"clientExtensions"' }],
-])
+export default defineMatrix(() => [allProviders, [{ previewFeatures: '' }, { previewFeatures: '"clientExtensions"' }]])
```

Comment:
The client extension preview feature can now be removed.

Reply:
Indeed, done!